lib/helpers/reports/convertStandups.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const _                   = require('underscore');
+const standupReport       = require('../getStandupReport');
+
+module.exports = function createAttachments(standups) {
+  let attachments = [];
+
+  _.each(standups, function (standup) {
+    attachments.push(standupReport(standup));
+  });
```

Comment:
I think I did an extraction on this. My preference is to do`Array.prototype.forEach`.
`
standups.forEach(function (standup) {
    attachments.push(standupReport(standup));
});
`